tests/components/lektrico/conftest.py

```diff
@@ -0,0 +1,101 @@
+"""Fixtures for Lektrico Charging Station integration tests."""
+
+from ipaddress import ip_address
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.zeroconf import ZeroconfServiceInfo
+
+MOCKED_DEVICE_IP_ADDRESS = "192.168.100.10"
+MOCKED_DEVICE_SERIAL_NUMBER = "500006"
+MOCKED_DEVICE_TYPE = "1p7k"
+MOCKED_DEVICE_BOARD_REV = "B"
+
+MOCKED_DEVICE_ZC_NAME = "Lektrico-1p7k-500006._http._tcp"
+MOCKED_DEVICE_ZC_TYPE = "_http._tcp.local."
+MOCKED_DEVICE_ZEROCONF_DATA = ZeroconfServiceInfo(
+    ip_address=ip_address(MOCKED_DEVICE_IP_ADDRESS),
+    ip_addresses=[ip_address(MOCKED_DEVICE_IP_ADDRESS)],
+    hostname=f"{MOCKED_DEVICE_ZC_NAME.lower()}.local.",
+    port=80,
+    type=MOCKED_DEVICE_ZC_TYPE,
+    name=MOCKED_DEVICE_ZC_NAME,
+    properties={
+        "id": "1p7k_500006",
+        "fw_id": "20230109-124642/v1.22-36-g56a3edd-develop-dirty",
+    },
+)
+
+
+def _mocked_device_config() -> dict[str, Any]:
+    return {
+        "type": MOCKED_DEVICE_TYPE,
+        "serial_number": MOCKED_DEVICE_SERIAL_NUMBER,
+        "board_revision": MOCKED_DEVICE_BOARD_REV,
+    }
+
+
+def _mocked_device_info() -> dict[str, Any]:
+    return {
+        "charger_state": "Available",
+        "charging_time": 0,
+        "instant_power": 0,
+        "session_energy": 0.0,
+        "temperature": 34.5,
+        "total_charged_energy": 0,
+        "install_current": 6,
+        "current_limit_reason": "Installation current",
+        "voltage_l1": 220.0,
+        "current_l1": 0.0,
+    }
+
+
+@pytest.fixture
+def mock_device():
+    """Patch Device class."""
+    with patch("homeassistant.components.lektrico.config_flow.Device") as MockDevice:
```

Comment:
Can you please double check AirGradient how we patch there?

Now we also have more fixtures, its a quite easy way to add tests for the rest of the integration using snapshot tests

Reply:
I changed the patch way. Thank you.